### PR TITLE
Note about computing rids

### LINKED ISSUE #30141: Clarify RID catalog casing
**Clarify RID catalog casing**

Developers sometimes need to compute RIDs programmatically, e.g. a custom publishing wizard in Visual Studio that can only rely information like the solution configuration manager and project properties. If the solution configuration passes invalid value, e.g. `ARM64` instead of `arm64`, you end up with a potentially invalid RID like `win-ARM64`.

We should clarify that computed RIDs must match the [catalog](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog). RIDs with different casing would also cause problems when the OS is case sensitive, like Linux, as the value is often used when constructing output paths, etc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 009616d0-6c93-a2de-82fe-4f7168817f18
* Version Independent ID: 4c2c5549-0151-516d-91d0-9b017c74e787
* Content: [.NET Runtime Identifier (RID) catalog](https://docs.microsoft.com/en-us/dotnet/core/rid-catalog)
* Content Source: [docs/core/rid-catalog.md](https://github.com/dotnet/docs/blob/main/docs/core/rid-catalog.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #30141.

### CODE REVIEW

File: docs/core/rid-catalog.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ RIDs that represent concrete operating systems usually follow this pattern: `[os
 
 - `[version]` is the operating system version in the form of a dot-separated (`.`) version number. For example, `15.10`.
 
-  - The version **shouldn't** be marketing versions, as they often represent multiple discrete versions of the operating system with varying platform API surface area.
+  The version **shouldn't** be a marketing version, as they often represent multiple discrete versions of the operating system with varying platform API surface area.
```
Comment (tdykstra) [inline docs/core/rid-catalog.md:29]: Changing versions to singular makes it out of sync with the following "they"

```suggestion
  The version **shouldn't** be a marketing version, as a marketing version often represents multiple discrete versions of the operating system with varying platform API surface area.
```

File: docs/core/rid-catalog.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ RID is short for *Runtime Identifier*. RID values are used to identify target pl
 They're used by .NET packages to represent platform-specific assets in NuGet packages. The following values are examples of RIDs: `linux-x64`, `ubuntu.14.04-x64`, `win7-x64`, or `osx.10.12-x64`.
 For the packages with native dependencies, the RID designates on which platforms the package can be restored.
 
-A single RID can be set in the `<RuntimeIdentifier>` element of your project file. Multiple RIDs can be defined as a semicolon-delimited list in the project file's `<RuntimeIdentifiers>` element. They're also used via the `--runtime` option with the following [.NET CLI commands](./tools/index.md):
+A single RID can be set in the [<RuntimeIdentifier>](project-sdk/msbuild-props.md#runtimeidentifier) element of your project file. Multiple RIDs can be defined as a semicolon-delimited list in the project file's [<RuntimeIdentifiers>](project-sdk/msbuild-props.md#runtimeidentifiers) element. They're also used via the `--runtime` option with the following [.NET CLI commands](./tools/index.md):
```
Comment (tdykstra) [inline docs/core/rid-catalog.md:13]: Code style won't be rendered but it keeps this from being interpreted as a html element. You could escape the open angle bracket instead.

```suggestion
A single RID can be set in the [`<RuntimeIdentifier>`](project-sdk/msbuild-props.md#runtimeidentifier) element of your project file. Multiple RIDs can be defined as a semicolon-delimited list in the project file's [<RuntimeIdentifiers>](project-sdk/msbuild-props.md#runtimeidentifiers) element. They're also used via the `--runtime` option with the following [.NET CLI commands](./tools/index.md):
```

**Review by tdykstra — APPROVED** [review]
LGTM, a couple of suggestions

File: docs/core/rid-catalog.md (hunk 1 above)
Comment (gewarren) [reply docs/core/rid-catalog.md:29]:
```suggestion
  The version **shouldn't** be a marketing version, as marketing versions often represent multiple discrete versions of the operating system with varying platform API surface area.
```

**Comment by gewarren** [comment]
Thanks @tdykstra!

**Pull request merged**